### PR TITLE
Add reviews

### PR DESCRIPTION
**Pull request by akshar-raaj**
* Added review on detail page.

* Setting hub_item_review.username using before_naming() hook.

reviews
skitch

### CODE REVIEW

File: hub/hub/doctype/hub_item_review/hub_item_review.py [hunk 1]
```diff
@@ -7,4 +7,8 @@
 from frappe.model.document import Document
 
 class HubItemReview(Document):
-	pass
+
+    def before_naming(self):
+        if self.user:
+            user = frappe.get_doc('User', self.user)
+            self.username = user.full_name or user.username or None
```
Comment (akshar-raaj) [inline hub/hub/doctype/hub_item_review/hub_item_review.py:14]: @rmehta @pratu16x7 What is a better way to achieve setting hub_item_review.username. I guess before_naming should only be used for setting `name`.

Comment (pratu16x7) [reply hub/hub/doctype/hub_item_review/hub_item_review.py:14]: A helper in the User doctype would do I feel, to `get_name()`

**Comment by akshar-raaj** [comment]
@rmehta @pratu16x7 Have added the following for company products and company detail.
company-logo
company-detail
company-list

**Comment by pratu16x7** [comment]
#### Sectioning
We can separate out the reviews section like the alternate sectioning on erpnext.org, by just adding a `section-bg` class to the section. Same goes for the featured items page.

#### Detail pages
For item details, let's make the image bigger (half the page width) and put it in a box. Alternate sectioning will take care of containment

#### Review listing
We can use custom list groups for the review listing: https://getbootstrap.com/docs/4.0/components/list-group/#custom-content

The media object of the current reviews covers everything, the individual layout can  look like Charles quote here: https://erpnext.org/fellows, smaller.

**Comment by akshar-raaj** [comment]
@pratu16x7 Have made the changes. Can you please review and merge.
item-box
reviews-list-group

**Review by pratu16x7 — APPROVED** [review]

**Pull request merged**